velox/exec/fuzzer/AggregationFuzzerBase.h

```diff
@@ -53,6 +53,24 @@ using facebook::velox::fuzzer::SignatureTemplate;
 
 constexpr const std::string_view kPlanNodeFileName = "plan_nodes";
 
+// For come functions, velox supports NaN, Infinity better than presto query
+// runner, which makes the comparison impossible.
+// Add data constraint in vector fuzzer to enforce to not generate such data for
+// those functions before they are fixed in presto query runner
+static const std::unordered_map<std::string, Constraint> functionDataConstrain =
```

Comment:
Move this to AggregationFuzzerTest.cpp and WindowFuzzerTest.cpp and pass it to AggregationFuzzer and WindowFuzzer. AggregationFuzzerBase needs to be general and not specific to Presto.

Reply:
Moved and only add it when we're using PrestoQueryRunner